.gitignore

```diff
@@ -1,4 +1,388 @@
-bin/
```

Comment:
Should we be deleting this? Is it included in the below?
Why add all this below? Can we just add small bits as we go, keep it simple and add complexity progressively?

Reply:
I usually use this file, which is the default Visual Studio template.
But whatever you see fit. I will do.
https://github.com/github/gitignore/blob/master/VisualStudio.gitignore

Reply:
If it’s ok let’s just add the things we use and not everything. I’m assuming they have this as a reference to pick and choose from.

Also I use VSCode and others may use other tools, so they can also add bits and it’s all discoverable and manageable.